core/templates/dev/head/components/summary_tile/CollectionSummaryTileDirective.js

```diff
@@ -34,17 +34,20 @@ oppia.directive('collectionSummaryTile', [
         getThumbnailIconUrl: '&thumbnailIconUrl',
         getThumbnailBgColor: '&thumbnailBgColor',
         isLinkedToEditorPage: '=?isLinkedToEditorPage',
-        getCategory: '&category'
+        getCategory: '&category',
+        isPlaylistMode: '&playlistMode',
+        showLearnerDashboardIcons: '&showLearnerDashboardIcons'
       },
       templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
         '/components/summary_tile/' +
         'collection_summary_tile_directive.html'),
       controller: [
         '$scope', 'oppiaDatetimeFormatter',
         'COLLECTION_VIEWER_URL', 'COLLECTION_EDITOR_URL', function($scope,
```

Comment:
Break line after '('. Consider indenting the stuff in '( ... )' by 2 extra spaces to distinguish it from the stuff that's actually in the inner scope.

Reply:
Done!